x/icacallbacks/icacallbacks.go

```diff
@@ -44,10 +44,13 @@ func ParseTxMsgData(acknowledgementResult []byte) ([][]byte, error) {
 }
 
 // UnpackAcknowledgementResponse unmarshals IBC Acknowledgements, determines the status of the acknowledgement (success or failure)
-//     and, if applicable, assembles the message responses
+//
+//	and, if applicable, assembles the message responses
+//
 // ICA transactions have associated messages responses. IBC transfer do not.
 // With ICA transactions, the schema of the response differs depending on the version of ibc-go used,
-//     however, this function unifies the format into a common response (a slice of byte arrays)
+//
+//	however, this function unifies the format into a common response (a slice of byte arrays)
```

Comment:
superrr nit and no need to revert, but whoever has these format settings should probably disable the setting that adds a new blank line